packages/integration-karma/test/events/focus-event-related-target/index.spec.js

```diff
@@ -1,6 +1,10 @@
 import { createElement } from 'lwc';
 import Container from 'x/container';
 
+// Note that focusin events are not fired in Chrome and Firefox if the browser content window is not focused.
+// Test for yourself: https://bl.ocks.org/nolanlawson/raw/d8ddc518041e9bf12498b8b50b39df95/
+// So this test may "flap" during local development if your browser window isn't focused (e.g. you're focused on
+// the DevTools instead).
```

Comment:
On top of this, I would recommend using [withContext](https://jasmine.github.io/api/3.7/matchers.html#withContext) to print some extra information when the test is failing.

```js
expect(elm.relatedTargetClassName)
        .withContext("This test may 'flap' during local development if your browser window isn't focused ")
        .toBe('first');
```

Output:

```
Chrome 90.0.4430.85 (Mac OS 10.15.7) events focus-event-related-target should retarget relatedTarget FAILED
	Error: This test may 'flap' during local development if your browser window isn't focused : Expected undefined to be 'first'.
	    at <Jasmine>
	    at UserContext.<anonymous> (events/focus-event-related-target/test/events/focus-event-related-target/index.spec.js:10:137 <- events/focus-event-related-target/index.spec.js:96:141)
	    at <Jasmine>
```

Reply:
Ahhhh, that's neat! TIL